schema/vrs.yaml

```diff
@@ -396,27 +387,68 @@ definitions:
     required: [ "type", "gene_id" ]
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # Range
+  # Numerics, Comparators, and Ranges
 
   Range:
     description: >-
-      An inclusive range of integers. A range may be unbounded by omitting either
-      min or max values. Fully unbounded ranges are not permitted (one of min or
-      max MUST be specified).
+      A set of classes that provide the possible representations of integer number
+      ranges, from a single integer (BoundedInteger) to bounded ranges (ClosedRange)
+      or half-bounded ranges (OpenRange).
+    oneOf:
+      - $ref: "#/definitions/BoundedInteger"
```

Comment:
An integer doesn't have bounds, so the meaning of BoundedInteger isn't transparent to me.